interface/wx/app.h

```diff
@@ -1061,6 +1061,23 @@ class wxApp : public wxAppConsole
     */
     static void GTKSuppressDiagnostics(int flags = -1);
 
+    /**
+        Allows wxWidgets to selectively suppress some GTK messages.
+
+        This function can be called to allow wxWidgets to control GTK message
+        logging. You must @e not call it if your application calls @c
+        g_log_set_writer_func() function itself, as this function can be only
+        called once.
+
+        It is recommended to call this function in your overridden version of
+        wxApp::OnInit() to allow wxWidgets to suppress some spurious GTK error
+        messages, e.g. the ones that happen whenever wxNotebook pages are
+        removed with the current GTK versions.
+
+        @since 3.2.1
```

Comment:
This should be marked as wxGTK only as well.

Reply:
Oops, thanks for noticing this, fixed (and the other one too).

I won't push the updated version yet to avoid rerunning the CI builds needlessly.